stripevpaypal.md

```diff
@@ -0,0 +1,115 @@
+
+## How to Set Up
+
+1. On Squarespace, navigate to *Commerce -> Payments*.
+2. Once there, PayPal and Stripe will show up as payment methods.
+3. For both, an account must be linked to use.
+4. To create an account for Stripe, navigate to [Stripe Registration](https://dashboard.stripe.com/register).
+5. To create an account for PayPal, navigate to [PayPal Business Signup](https://www.paypal.com/bizsignup).
+6. Once an account is created, link the account on Squarespace and it will be ready for use.
+
+## Stripe and PayPal Payment Comparison on Squarespace
+
+| Aspect                       | PayPal                                                                                                    | Stripe                                                                                     |
+|------------------------------|-----------------------------------------------------------------------------------------------------------|--------------------------------------------------------------------------------------------|
+| **Ease of Use**              | Squarespace makes integration simple. Users can use existing or set up a new account through Squarespace. | Seamless integration with just a Stripe account.                                           |
+| **Transaction Fees**         | Varies by user's location.                                                                                | Varies by country.                                                                         |
+| **Processing**               | Allows checkout without sharing financial info on the merchant site. Payment occurs on PayPal’s website.  | Payments take place on the merchant site with Stripe, providing a simpler user experience. |
+| **Accepted Payment Methods** | PayPal Balance, Visa, Mastercard, American Express, Discover, PayPal Credit.                              | Major credit and debit cards, Apple Pay, Google Pay.                                       |
+| **Security**                 | Secure transactions processed off the merchant site with strong buyer protection policies.                | Strong emphasis on security, using encryption to secure transactions and user data.        |
+| **Support**                  | Phone, email, help center for support.                                                                    | Email support, live support for high-tier plans.                                           |
+
+## Pros and Cons of Integrating PayPal on Squarespace
+
+### Pros:  
+**Trust and Recognition**  
+PayPal is one of the most recognized online payment systems. Customers often trust PayPal and are familiar with its payment process.
+*Source:* [PayPal's About Us page](https://www.paypal.com/us/webapps/mpp/about)  

```

Comment:
I don't think the sources are necessary in this article.

Reply:
Removed line 36 source